src/phantom.js

```diff
@@ -75,26 +75,15 @@ export default class Phantom {
         this.process.on('exit', code => logger.debug(`Child exited with code {${code}}`));
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
-	        logger.error(error);
-	        console.log(error);
-            this.process.exit(1);
+            logger.error(error);
+            this.kill();
+            process.exit(1);
```

Comment:
that's what it was originally if you look at this repo. I don't know why that was there honestly. Seems like it shouldn't have been.